app/adBlock.js

```diff
@@ -10,6 +10,18 @@ const ABPFilterParser = ABPFilterParserLib.ABPFilterParser
 const FilterOptions = ABPFilterParserLib.FilterOptions
 const DataFile = require('./dataFile')
 const Filtering = require('./filtering')
+const appConfig = require('../js/constants/appConfig')
+const debounce = require('../js/lib/debounce.js')
```

Comment:
doh that was from copy/paste, I'll get rid of all files at once when I fix this to avoid that